Option for antenna with non-zero velocity (#2832)
ECP-WarpX/WarpX: Docs/source/usage/parameters.rst

    per angular mode. The laser particles are loaded into radial spokes, with
    the number of spokes given by min_particles_per_mode*(warpx.n_rz_azimuthal_modes-1).

> * ``<laser_name>.v_antenna`` (list of `float`) optional (defaut `0. 0. 0.`)

Reviewer: ```suggestion
* ``<laser_name>.v_antenna`` (list of `float`) optional (default `0. 0. 0.`)
```

>     Velocity of the moving antenna, in unit of speed of light. For now, can only be non-zero in the z direction.

Reviewer: I had to read this twice: I think that can cause a little misunderstanding between:
- all axes must be zero but z can be different (what we man) and
- cannot be zero in the z direction (which we don't mean)

Maybe that's clearer?
```suggestion
    Velocity of the moving antenna, in unit of speed of light. For now, an antenna can only move in ``z``.
```
